src/app/not-found.tsx

```diff
@@ -1,5 +1,12 @@
 const NotFound = () => {
-  return <div>404 Page Not Found</div>;
+  return (
+    <div className="flex h-screen items-center justify-center">
+      <p className="text-4xl font-normal">
```

Comment:
```suggestion
      <p className="text-4xl">
```
remove font-normal it does nothing. the default font of the repository is already set to montserrat so you don't need to change the font unless your text uses the other font, "the seasons" which you would use font-seasons